Increase the limit for Proto size

pengyaoc opened the pull request:
Summary: ParseProtoFromLargeString hits limit when using recurring v2. To unblock warmup project, we can increase the limit temporarily. More details in this post -- https://fb.facebook.com/groups/264913123977784/permalink/463566404112454/

Differential Revision: D9436368

Yangqing commented:
Thanks - probably easy to just increase that to 2G (aka, hard limit)? https://stackoverflow.com/questions/34128872/google-protobuf-maximum-size

Yangqing commented:
coded_stream uses default 64MB limit, so you have to have the SetTotalBytesLimit function there. We can do largest 32-bit interger there indeed :)

Commit:
Increase the limit for Proto size (#10745)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/10745

ParseProtoFromLargeString hits limit when using recurring v2. To unblock warmup project, we can increase the limit temporarily. More details in this post -- https://fb.facebook.com/groups/264913123977784/permalink/463566404112454/

Differential Revision: D9436368

fbshipit-source-id: c91de883c3f6eb3ec3b5194219d5ae8217915cdd
